app/jobs/check_consolidated_screening_list_job.rb

```diff
@@ -7,8 +7,8 @@ def perform(user)
     ScoutApm::Transaction.ignore! if rand > (ENV["SCOUT_SAMPLE_RATE"] || 1).to_f
     @user = user
     return unless @user
-
-    return if ImportExport::Client.new.search(name: @user.name).empty?
+    client = ConsolidatedScreeningList::Client.new api_key: ENV["TRADE_API_KEY"]
```

Comment:
This environment variable may be a little too generic. Thoughts?

Reply:
This is what they recommend in their docs. Its the api key for Trade.gov so I think we should keep it.